scripts/create-kubestellar-demo-env.sh

```diff
@@ -220,18 +266,18 @@ helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart
         --set-json='verbosity.default=5' \
         --set-json='kubeflex-operator.verbosity=5' \
         --timeout=24h \
-        $var_flags
+        "$var_flags"
```

Comment:
We do _not_ want dquotes around this variable expansion. We want it to be followed by word splitting.  See https://www.gnu.org/software/bash/manual/bash.html#Shell-Expansions-1